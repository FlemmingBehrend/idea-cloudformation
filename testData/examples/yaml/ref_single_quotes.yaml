AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  ECSCluster:
    Type: String
    Description: Name of an existing ECS Cluster.
Resources:
  service:
    Type: AWS::ECS::Service
    Properties:
      Cluster: !Ref '<caret>
      DesiredCount: '1'
      TaskDefinition: AA