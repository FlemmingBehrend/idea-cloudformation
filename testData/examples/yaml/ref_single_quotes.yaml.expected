AWSTemplateFormatVersion: '2010-09-09'
Parameters:
  ECSCluster:
    Type: String
    Description: Name of an existing ECS Cluster.
Resources:
  service:
    Type: AWS::ECS::Service
    1@<Properties:>
      Cluster: 0@<!Ref '
      DesiredCount: '1'>
      TaskDefinition: AA

0: Too many values
1: Required resource properties are not set: DesiredCount
